docker/README.md

```diff
@@ -1,14 +1,15 @@
 ### Building Redox using Docker images with the toolchain
 
-*All you need is git, make, qemu, fuse and docker. The method requires a non-privileged user able to run the `docker` command, which is usually achieved by adding the user to the `docker` group.*
+*All you need is rustup, git, make, qemu, fuse and docker. The method requires a non-privileged user able to run the `docker` command, which is usually achieved by adding the user to the `docker` group.*
```

Comment:
You're right, we don't need the submodules to build and run the image, so we can just do `make pull all` instead of `make all` when we run the container.
The only problem is that `make pull` calls `make clean`, so every build will be a complete rebuild. I think the README should mention that `pull` could be omitted for subsequent builds.